src/components/Button/__snapshots__/Button.story.storyshot

```diff
@@ -122,6 +122,7 @@ exports[`Storybook Snapshot tests and console checks Storyshots Watson IoT/Butto
         aria-hidden="true"
         aria-label="Button icon"
         className="bx--btn__icon"
+        fill="currentColor"
```

Comment:
Is this a new carbon variable?

Reply:
Nope! https://css-tricks.com/currentcolor/